topics/contributing/faqs/where-to-report.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Posting issues and ideas for the Galaxy Community"
+area: github
+box_type: tip
+layout: faq
+contributors: [nomadscientist]
+---
+
+The Galaxy community addresses issues and needs through resolving issues on Github.
+
+# Are you struggling to analyse something and need help?
+
+{% snippet faqs/galaxy/analysis_reporting_issues.md %}
+
+# Have you found a key problem and/or had an idea that could improve Galaxy?
+
+Report an **Issue** on the correct Github repository:
+
+ - **Tools**: Need a tool added to a server? Check out the FAQ for this:
+      {% snippet faqs/galaxy/request-galaxy-tools-on-a-specific-server.md %}
+ - **Tools**: Bug in the tool: Select your tool in the Galaxy interface | Drop-down arrow to **See in Tool Shed** | **Development repository** , then describe the issue there
```

Comment:
For problems with tools, we should also mention the bug report feature on Galaxy itself, e.g. when you get a red dataset you can submit a bug report via the bug icon. This gives Galaxy admins the most details and context for what is going wrong.

So unless you know for sure it is a problem with the wrapper (e.g. a parameter you want to use is missing), bug report is probably the thing you want (and easier to do for a user)

Reply:
ok I guess that is what you mention under "struggling to analyse" already, nvm